### DISCUSSION TITLE
Auth with Service Connection does not have Permissions in Tenant..

### DISCUSSION THREAD

**Discussion by TimFeiler** [microsoft/powerplatform-build-tools#653] [Q&A]
Hello everyone,

I have registered an authentication profile with a Service Connection. The Service Connection has been added to Dataverse as an App User with Admin permissions however it does not seem to have permissions on any orgs. Org list comes back empty, connections create ends up with:     Der Remotename konnte nicht aufgelöst werden: '000000000000000000000000000000.00.environment.api.powerplatform.com'
![image](https://github.com/microsoft/powerplatform-build-tools/assets/74839878/fdc81546-f6c1-45bc-8fa5-97a74bc77694)
Also changing UAT to the actual environmentID when selecting an or did not help. 

What is it that I am missing?

**Comment by larmccoy** [comment]
I've been having similar issues.  I was able to create the connection when I provided the environment GUID.  I am still seeing issues with "pac org list" where I get a blank list.  When I enter "pac org who", I DO get the correction information for the current Org/Environment with which I'm working, so it seems to be connected to the right environment/org.

Also, I've found that "it all works better" when I create two authorization profiles for the service principal, one that is DATAVERSE and one that is UNIVERSAL.  There are some behind-the-scenes inconsistencies in the CLI currently that are being worked out, where the "KIND" of auth profile seems to determine your level of success with certain commands (especially "pac connection" and "pac org")
